.github/workflows/chart.yaml

```diff
@@ -34,6 +36,36 @@ jobs:
         uses: azure/setup-helm@v1
         with:
           version: v3.7.1
+      - name: Set chart output
+        id: chart
+        run: |
+          CHART=$(find . -type f  -name "elemental-operator*.tgz" -print)
+          echo "::set-output name=chart_name::$CHART"
+      - name: Test chart values
+        if: ${{ !startsWith(github.ref, 'refs/tags/') }}
+        run: |
+          REPO=$(helm show values ${{ steps.chart.outputs.chart_name }}|yq .image.repository)
+          TAG=$(helm show values ${{ steps.chart.outputs.chart_name }}|yq .image.tag)
+          if [ "$REPO" != "quay.io/costoolkit/elemental-operator-ci" ];then
+            echo "REPO on the built chart is not the expected value"
+            echo "Expected quay.io/costoolkit/elemental-operator but got $REPO"
+            exit 1
+          fi
+      - name: Test release chart values
+        if: startsWith(github.ref, 'refs/tags/')
+        run: |
+          REPO=$(helm show values ${{ steps.chart.outputs.chart_name }}|yq .image.repository)
+          TAG=$(helm show values ${{ steps.chart.outputs.chart_name }}|yq .image.tag)
+          if [ "$REPO" != "quay.io/costoolkit/elemental-operator" ];then
```

Comment:
It seems that `quay.io/costoolkit/elemental-operator` is empty, are we going to build there images

Reply:
yep, on release, images tagged with a version should be pushed there, unfortunately we havent cut a release yet :D